stock_picking_invoice_link/__manifest__.py

```diff
@@ -1,37 +1,25 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Copyright (C) 2013-15 Agile Business Group sagl (<http://www.agilebg.com>)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2013-17 Agile Business Group (<http://www.agilebg.com>)
+# © 2016 AvanzOSC (<http://www.avanzosc.es>)
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2017 Jacques-Etienne Baudoux <je@bcim.be>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
     'name': 'Stock Picking Invoice Link',
-    'version': '8.0.1.0.0',
+    'version': '10.0.0.2.0',
```

Comment:
Why this version number? Follow conventions: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

Reply:
@chienandalu see https://github.com/OCA/multi-company/pull/57#discussion_r136263566